buildlib/pr/codestyle.yml

```diff
@@ -90,3 +90,21 @@ jobs:
           pip3 install codespell
           codespell
         displayName: codespell test
+
+  - job: ctags_generation
+    displayName: ctags check
+    pool:
+      name: MLNX
+      demands:
+      - ucx_docker -equals yes
+    container: fedora
+    steps:
+      - checkout: self
+        clean: true
+        fetchDepth: 100
+        retryCountOnTaskFailure: 5
+
+      - bash: |
+          sudo yum -y install ctags vim
```

Comment:
can we add them to the docker image? or use a docker image that has them?

Reply:
Created request to add the two binary to the application.
Internal: [2731](https://jirasw.nvidia.com/browse/HPCINFRA-2731)

Reply:
fixed and pushed

Reply:
never mind, image was restored with current cppcheck 2.3 version and corresponding CI now passes.